Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesDialog.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.commonfilesearch;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JFrame;
+import org.openide.util.NbBundle;
+import org.openide.windows.WindowManager;
+
+/**
+ *
+ * @author bsweeney
+ */
```

Comment:
We do not do these types of class headers, please remove authorship and provide description.